templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -218,21 +189,58 @@ <h3 class="p-heading--5">Conformity assessment</h3>
         </div>
       </div>
     </div>
-    <div class="p-section--shallow">
-      {{ image (
-      url="https://assets.ubuntu.com/v1/590335cc-servers.png",
-      alt="",
-      width="2464",
-      height="1028",
-      hi_def=True,
-      loading="lazy"
-      ) | safe
-      }}
-    </div>
   </div>
 
   <hr class="p-rule is-fixed-width" />
 
+  <div class="u-fixed-width">
+    <section class="p-section">
+      <div class="row--50-50">
+        <div class="col">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/c46635ff-what-products-and-devices.png",
+          alt="",
+          width="1200",
+          height="1801",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+          }}
+        </div>
+        <div class="col">
+          <h2>
+            What products and devices
+            <br class="u-hide--small u-hide--medium" />
+            does the CRA regulate?
+          </h2>
+          <p>
+            The CRA will regulate all manufacturers and developers who want to put products on the EU market, regardless of their location. The CRA will cover all products or devices available in the EU market that are connected to any other device or network to exchange data. This includes PDEs that use:
+          </p>
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked u-hide"></li>
```

Comment:
Instead of creating and hiding a list item, we can use  a `hr` tag with margin/padding removed to get similar result